### PR TITLE
Pd 318 backport index

### PR DESCRIPTION
**Pull request by catherine-jones**
update index.html and index.js with fixes implemented in productotype.

### CODE REVIEW

File: lib/assets/src/scripts/index.js [hunk 1]
```diff
@@ -80,9 +80,12 @@ d3.selectAll('.bar .widget__inner').each(function() {
   let dataRaw = widget.datasets;
```
Comment (elisechant) [inline lib/assets/src/scripts/index.js]: @catherine-jones we need to roll out changes to index.js

File: app/views/dashboards/index.html.erb [hunk 2]
```diff
@@ -2,26 +2,22 @@
...
   <ul class="list-unstyled clearfix">
     <% dashboards.each do |dashboard| %>
-      <li class="item--australian-citizenship-appointment-booking-service-dashboard card row">
+      <li class="card row"> <!--TODO add helper to generate class based on dashboard service name-->
         <div class="card__left">
           <div>
-            <h2 class="card-title"><%= dashboard.name %></h2>
+            <h2 class="card-title"><%= dashboard.name %></h2> <!--TODO add helper to ensure dashboard is on the end of the name-->
             <div class="updated-at">
               <p>Updated at <%= dashboard.published_at.strftime("%d %b %Y %H:%M:%S") %></p>
```
Comment (elisechant) [inline app/views/dashboards/index.html.erb]: are we able to store this transformation in a helper?

Comment (tobyhede) [inline app/views/dashboards/index.html.erb]: Yes. We actually have this whole decorator pattern lib that we can use to. `dashboard_decorator.rb`

Comment (elisechant) [inline app/views/dashboards/index.html.erb]: real nice 👍. let's break this off then from this specific task. LGTM

**Comment by elisechant** [comment]
@catherine-jones we should also port the widget partials from that Index route as well? if it's separate task, maybe we need a new ticket for that task?

**Comment by catherine-jones** [comment]
@elisechant yeah i think break it out into another ticket would be good

**Pull request merged**